docs/netdata-agent/versions-and-platforms.md

```diff
@@ -67,3 +67,49 @@ Static builds usually miss certain features that require operating-system suppor
 - eBPF related features
 
 When platforms are removed from the [Binary Distribution Packages](/packaging/makeself/README.md) list, they default to install or update Netdata to a static build. This may mean that after platforms become EOL, Netdata on them may lose some of its features. We recommend upgrading the operating system before it becomes EOL, to continue using all the features of Netdata.
+
+### Upgrading on an Unsupported Platform
+
+Existing installs of native packages on a platform that is no longer supported, cannot be automatically updated at this time. The upgrader will show it is already at the newest version, because we no longer publish new packages.
```

Comment:
```suggestion
Existing installs of native packages on a platform that is no longer supported, cannot be automatically updated. The upgrade process will show it is already at the newest version, because we no longer publish new packages for that platform.
```

Reply:
I put in "at this time", because we may want to automate it later (and then make it a supported operation).

Reply:
Users don't need to worry/ be informed about this though

Reply:
In that case I will change it to "are not automatically updated".